tools/src/generator.ts

```diff
@@ -162,6 +173,9 @@ export function mapWidget(
   const subscribableOptions: ISubscribableOption[] = collectSubscribableRecursively(raw.options)
     .map(mapSubscribableOption);
 
+  const freeFunctions: IFreeFunctions[] = collectFreeFunctions(raw.options)
+    .map((o:IProp): IFreeFunctions => ({ name: o.name }));
```

Comment:
```suggestion
    .map(({name}:IProp): IFreeFunctions => ({ name }));
```

it is better to move this map into collectFreeFunctions